src/lib/custom-helpers.ts

```diff
@@ -8,7 +8,11 @@ import { CommandMessage } from 'discord.js-commando';
  * @returns {number}
  */
 export function getEmbedColor(msg: CommandMessage): number {
-	let embedColor: number = parseInt(msg.client.provider.get(msg.guild.id, 'embedColor', '555555'), 16);
+	let embedColor: number = parseInt('555555', 16);
+	if (msg.guild) {
+		embedColor = parseInt(msg.client.provider.get(msg.guild.id, 'embedColor', '555555'), 16);
+	} else {
+	}
 
 	// This shouldn't happen, but if it does, return the default embed color
 	if (embedColor > parseInt('FFFFFF', 16)) {
```

Comment:
Can we add a check to make sure that the `embedColor` is between `#000000` and `#FFFFFF`?

```suggestion
	if (embedColor > parseInt('FFFFFF', 16) || embedColor < 0) {
```